include/avif/internal.h

```diff
@@ -89,6 +89,10 @@ void avifImageSetDefaults(avifImage * image);
 // Copies all fields that do not need to be freed/allocated from srcImage to dstImage.
 void avifImageCopyNoAlloc(avifImage * dstImage, const avifImage * srcImage);
 
+// Copies the samples from srcImage to dstImage. dstImage must be allocated.
+// srcImage and dstImage must have the same width, height, depth, yuvFormat and yuvRange.
```

Comment:
I found that when copying only the alpha plane (`planes == AVIF_PLANES_A`), `yuvFormat` may be set to `yuvFormat = AVIF_PIXEL_FORMAT_NONE`. So the second sentence should be shortened to:

```
srcImage and dstImage must have the same width, height, and depth.
```

I think we can add a third sentence:

```
If the AVIF_PLANES_YUV bit is set in `planes`, then srcImage and dstImage must have the same yuvFormat and yuvRange.
```